### PR TITLE
MM-T260 Mark as Unread New Messages line extra space moves with it

### PR DESCRIPTION
**Pull request by yulyanaR**
Summary

Cypress test: Mark as Unread New Messages line extra space moves with it MM-T260

Ticket Link

https://automation-test-cases.vercel.app/test/MM-T260

### CODE REVIEW

File: e2e/cypress/integration/mark_as_unread/mark_as_unread_spec.js [hunk 1]
```diff
@@ -131,15 +131,33 @@ describe('Mark as Unread', () => {
         // The New Messages line should appear above the selected post
         verifyPostNextToNewMessageSeparator('post2');
```
Comment (saturninoabril) [inline e2e/cypress/integration/mark_as_unread/mark_as_unread_spec.js:133]: I'm fine to modify this test case. With that, please change the title as follows.
```diff
- it('Should be able to mark channel as unread by alt-clicking', () => {
+ it('MM-T260 Mark as Unread New Messages line extra space moves with it', () => {

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @yulyanaR! Looks good to me, except for one request.

**Review by Willyfrog — APPROVED** [review]
LGTM!

File: e2e/cypress/integration/mark_as_unread/mark_as_unread_spec.js (hunk 1 above)
Comment (yulyanaR) [reply e2e/cypress/integration/mark_as_unread/mark_as_unread_spec.js:133]: Thank you, it's been updated.

**Review by saturninoabril — APPROVED** [review]
Thanks @Yulyana-R!

**Comment by prapti** [comment]
In test queue.

**Review by prapti — APPROVED** [review]
Great work on this test work @yulyanaR! Thank you again for this contribution 🎉

**Pull request merged**